java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -157,6 +162,8 @@ public static Object[] transactionTestResult() {
             Map.of("timeout", "1000"),
             OK,
             "GLIDE", // echo
+            0L, // rpushx(listKey3, new String[] { "_" })
```

Comment:
Why not push to an already existing list or will this be changed later.

Reply:
I can't track content of existing lists
I hope transaction rework would help there